purchase_reception_notify/models/stock_picking.py

```diff
@@ -0,0 +1,49 @@
+# Copyright 2019 ForgeFlow S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/lgpl-3.0).
+
+from odoo import _, api, models
+
+
+class StockPicking(models.Model):
+    _inherit = "stock.picking"
+
+    @api.model
+    def _purchase_order_picking_confirm_message_content(self, picking, purchase_dict):
+        if not purchase_dict:
+            purchase_dict = {}
+        title = _("Receipt confirmation %s") % (picking.name)
+        message = "<h3>%s</h3>" % title
+        message += _(
+            "The following items have now been received in Incoming Shipment %s:"
+        ) % (picking.name)
+        message += "<ul>"
+        for purchase_line_id in purchase_dict.values():
+            message += _("<li><b>%s</b>: Received quantity %s %s</li>") % (
+                purchase_line_id["purchase_line"].product_id.display_name,
+                purchase_line_id["stock_move"].product_qty,
+                purchase_line_id["stock_move"].product_uom.name,
+            )
+        message += "</ul>"
+        return message
+
+    def _action_done(self):
+        super(StockPicking,self)._action_done()
```

Comment:
Precommit failing due to a missing space, you could add the space or do this:
```suggestion
        super()._action_done()
```

Reply:
Run pre-commit again! there is another issue here: https://github.com/OCA/purchase-workflow/pull/1174/checks?check_run_id=2607277868#step:4:105